### PR TITLE
small fixup of README dependencies

### PR DESCRIPTION
**Pull request by kbhawkey**
- Updated a code block
- Cleaned up the command line build dependencies to match theme instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +45,13 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
+Install dependencies:
+
+```
+sudo npm install -D --save autoprefixer
+sudo npm install -D --save postcss-cli
```
Comment (sftim) [inline README.md:52]: This doesn't look right - we don't want to insist that contributors run commands as the superuser if we can avoid it.

Comment (sftim) [reply README.md:52]: Can we skip the `sudo`?

**Review by sftim — COMMENTED** [review]
A query about the need for `sudo`

File: README.md (hunk 1 above)
Comment (kbhawkey) [reply README.md:52]: OK. I pulled the commands from the theme's README.

**Comment by kbhawkey** [comment]
Recent updates cover these changes.
/close

**Pull request closed without merging**